apps/daimo-mobile/src/view/screen/DepositScreen.tsx

```diff
@@ -92,7 +95,8 @@ function OnrampStub() {
     <View style={styles.callout}>
       <TextBody>
         <Octicons name="alert" size={16} color="black" />{" "}
-        <TextBold>Onramp coming soon.</TextBold> You'll be able to buy USDC
+        <TextBold>Onramp coming soon.</TextBold> You'll be able to buy $
+        {tokenMetadata.symbol}
```

Comment:
don't need the $, this will render as `$USDC`

Reply:
good catch, fixed!